### PR TITLE
Fixes - Cannot override frozen service 'tpl' - (DIC) issue

### PR DESCRIPTION
**Pull request by tomaszkolonko**
This issue led to errors when a user (opposed to an admin) tried to open a LiveVoting instance. Inside the browser the message was: "ERR_TOO_MANY_REDIRECTS". In the apache2 error logs the message was: "AH01071: Got error 'PHP message: Cannot override frozen service "tpl".\n'".

The 'tpl' DIC instance is instantiated several times. Commenting out the parent::initHTML() solved the issue.

It was tested on our side.

### CODE REVIEW

**Comment by rsheer** [comment]
We had the same issue with our LiveVoting. The reason is the following:
In xlvoInitialisation on line 55, the file "inc.header.php" is included with a require statement:

    require_once('./include/inc.header.php');

This is not a classfile, but a script which directly executes code to initialise ILIAS (with a call to _ilInitialisation::initILIAS()_). On the next line, _initHTML2()_ is called, which tries at first to initialise HTML with the following line:

    parent::initHTML()  // <- this is a call to ilInitialisation::initHTML() since xlvoInitialisation is a child class

Since "_inc.header.php_" already initialized HTML (and with this the "tpl"), this leads to an error.

On our instance of ILIAS, the error leads to a redirect to the "error.php"-File in the LiveVoting directory. This file initializes ILIAS the same way as before which leads again to an error. The result: After about 30 redirects, the browser stops and prints out a redirect-error.

So long story short: The commit f4ad0db worked for us.

**Comment by mstuder** [comment]
Thx for you work!

Can you please check this in LiveVoting 4.1.0? If the issue still exists, Write us https://plugins.studer-raimann.ch/goto.php?target=uihk_srsu_PLLV
